docs/src/modules/java/pages/workflows.adoc

```diff
@@ -51,7 +51,7 @@ include::example$java-spring-transfer-workflow/src/main/java/com/example/wallet/
 <3> An endpoint to deposit funds to the wallet.
 <4> An endpoint to get current wallet balance.
 
-Now we can focus on the workflow implementation itself. A workflow has state, which can be updated in command handlers and transition methods. During the state modeling we might consider the information is required for validation, running the steps, collecting data from steps or tracking the workflow progress.
+Now we can focus on the workflow implementation itself. A workflow has state, which can be updated in command handlers and transition methods. During the state modeling we might consider the information that is required for validation, running the steps, collecting data from steps or tracking the workflow progress.
```

Comment:
Referring here to `transition methods` but not mentioned before. Maybe adding a link?

Reply:
changed to `steps implementation`